packages/@lwc/engine/scripts/jest/preprocessor.js

```diff
@@ -0,0 +1,27 @@
+/*
```

Comment:
Is there a need to creating a special jest transform if you just invoke babel as is.
Can't you move this to a local `babel.config.js` instead?